internal/receiver/databricksreceiver/spark_service_test.go

```diff
@@ -0,0 +1,100 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package databricksreceiver
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+
+	"github.com/signalfx/splunk-otel-collector/internal/receiver/databricksreceiver/internal/spark"
+)
+
+func TestSparkService(t *testing.T) {
+	ssvc := newTestSparkService()
+	metrics, err := ssvc.getSparkCoreMetricsForAllClusters()
+	require.NoError(t, err)
+	for _, metric := range metrics {
+		fmt.Printf("->%v<-\n", metric)
+	}
+}
+
+func xTestSparkService_Integration(t *testing.T) {
+	logger := zap.NewNop()
+	dbcl := newDatabricksClient(
+		"https://adb-4429673989716691.11.azuredatabricks.net",
+		"dapi29cab1150cc2f1eb365120abbb95d8da",
```

Comment:
please remove from code

Reply:
This is a draft PR. There are several things that need to be cleaned up before it's ready for review.